server/athenian/api/models/web/base_model_.py

```diff
@@ -138,6 +138,9 @@ def __init__(cls, name, bases, dikt):
         """Initialize a new enumeration class type."""
         super().__init__(name, bases, dikt)
         cls.__members = set(v for k, v in dikt.items() if not k.startswith("__"))
+        cls.__members.update(
+            chain.from_iterable(getattr(base, "_Enum__members", ()) for base in bases),
```

Comment:
What if the base name is not "Enum"? It will be named `_BaseName__members`...

Reply:
since `__members` is only defined in Enum class definition it will always called this way, even if code is run by a subclass; so this works:
```
In [1]: from athenian.api.models.web.base_model_ import Enum

In [2]: class A(metaclass=Enum):
   ...:     a = '1'
   ...: class B(A):
   ...:     b = '2'
   ...: class C(B):
   ...:     c = '3'
   ...: list(C)
   ...: 
Out[2]: ['3', '2', '1']
```